moveit_planners/chomp/chomp_motion_planner/src/chomp_optimizer.cpp

```diff
@@ -48,7 +48,11 @@ namespace chomp
 {
 double getRandomDouble()
 {
+#ifdef _WIN32
+  return ((double)rand() / (double)RAND_MAX);
+#else
   return ((double)random() / (double)RAND_MAX);
+#endif
```

Comment:
Aside from the fact that this code should probably use C++ `<random>` instead of global random functions at all,
is there any reason why you want the `ifdef` at all over using `rand` for all systems?

Reply:
No, I don't have any particular reason to stick with `rand()`. I changed the code to use `<random>` now.